packages/gatsby/src/gatsby-browser.js

```diff
@@ -0,0 +1,20 @@
+exports.onClientEntry = function(_, pluginParams) {
+  require.ensure(['@sentry/browser', '@sentry/apm'], function(require) {
+    const Sentry = require('@sentry/browser');
+    const TracingIntegration = require('@sentry/apm').Integrations.Tracing;
+    const tracesSampleRate = pluginParams.tracesSampleRate !== undefined ? pluginParams.tracesSampleRate : 0;
+    const integrations = [...(pluginParams.integrations || [])];
+    if (tracesSampleRate) {
+      integrations.push(new TracingIntegration());
+    }
+    Sentry.init({
+      environment: process.env.NODE_ENV || 'development',
+      release: __SENTRY_RELEASE__,
```

Comment:
@dcramer So you don't use `process.env` on these other environment variables? Strange...

Reply:
So Gatsby will only expose env variables with GATSBY_ prefixed to the frontend. NODE_ENV is special cased there IIRC. This is basically a shitty hack to force them to be exposed.